demo/angularjs/components/search/doc.md

```diff
@@ -0,0 +1,17 @@
+# Search
```

Comment:
I think this component should be named 
```suggestion
# Search field
```
Search is too much of a concept to not be accompanied by another name.